documentation/DevelopmentProcess/DevelopmentProcess.tex

```diff
@@ -181,7 +181,14 @@ \subsection{Versioning Scheme}
 Whenever an item needs to have versions, it will follow the \textit{major.minor.patch} versioning scheme. This means when anything major has changed, such as a feature being added or response to a specification being changed it will increment the current \textit{major} number by one. \textit{Minor} updates will be for changing the item to refine a feature or to meet a specification and will also be incremented by one. For any bug fixes or commentary changes, the \textit{patch} can be incremented by one.
 
 \section{Process Workflow}
-In order to facilitate the development of the system, work will be organized into 2-week long sessions. At the beginning of the week, the team will consider the next 2-3 deliverables and schedule them in accordingly. The team will attempt to finish each deliverable a week or more before the deadline. This allows the team to be ahead of schedule and gives them the ability to respond to other items should they arise. During meetings before the end of each 2-week session, any priorities will be adjusted as needed. When the work session is completed, items will be placed into the next session or placed in the backlog until the further consideration.
+In order to facilitate the development of the system, work will be organized into 2-week long sessions. At the beginning of the week, the team will consider the next 2-3 deliverables and schedule them in accordingly. The team will attempt to finish each deliverable a week or more before the deadline. This allows the team to be ahead of schedule and gives them the ability to respond to other items should they arise. During meetings before the end of each 2-week session, any priorities will be adjusted as needed. When the work session is completed, items will be placed into the next session or placed in the backlog until further consideration. \\\\
+For the hardware design and assembly of the system, the team is currently using a rapid prototyping approach similar to the cycle in Figure \ref{fig:rapid}. This allows the team to quickly develop and build prototypes for parts of the system. Instead of building everything at once, portions of the system will be built incrementally and integrated into each other to form our complete system. We are also utilizing the agile methodology in conjunction with rapid prototyping to allow sections of the system to be designed and built simultaneously. This approach is beneficial because it gives more opportunities to obtain feedback from TAs or Dr. Alan Wassyng to further refine the prototype. The team is using ready made parts and 3D printing for prototype development because it allows for quick changes and improvements to the prototype which will eventually be implement into the final system.
+\begin{figure}
```

Comment:
Looking at the figure, can we remove the "refine and iterate" block? The review + feedback loop covers that, so I don't think it is necessary.